db/data/20220308163700_add_ooda_fund_activity.rb

```diff
@@ -0,0 +1,26 @@
+# Add the top level (A) Other ODA Activity (i.e. a fund-tier activity)
+
+beis = Organisation.service_owner
+
+description = <<~TXT.squish
+  Other ODA represents spending from ‘other non ODA budgets from BEIS’ which may be
+  classed as ‘ODA spend’ i.e. is ODA-eligible in the same way as spend from the
+  Newton Fund and GCRF.
+TXT
+
+Activity.create!(
+  source_fund_code: 3,
+  roda_identifier: "OODA",
+  title: "Other ODA",
+  organisation: beis,
+  level: "fund",
+  form_state: "complete", # wizard progress
+  fstc_applies: false, # free_standing_technical_cooperation: false,
+  delivery_partner_identifier: "OODA",
+  description: description,
+  sector_category: "998", # 510, 998 for NF/GCRF (based on GCRF)
```

Comment:
Given that the sector and category have been confirmed with BEIS, I'm not sure the comments here add anything.